.gitignore

```diff
@@ -84,7 +82,6 @@
 
 # avoid issues when switching branches during transistion from old to new structure
 /src/
-/msfs-avionics-mirror/
```

Comment:
Can I suggest leaving all these gitignores for a few weeks at least to facilitate moving between and rebasing branches?